src/3box.js

```diff
@@ -143,9 +151,40 @@ class Box {
    * @param     {Object}    opts.ipfs               A js-ipfs ipfs object
    * @param     {String}    opts.orbitPath          A custom path for orbitdb storage
    * @param     {Boolean}   opts.iframeStore        Use iframe for storage, allows shared store across domains. Default true when run in browser.
+   * @param     {Boolean}   opts.api                Use 3Box API to fetch profile instead of OrbitDB. Default true.
```

Comment:
Maybe this param is more clear if it's called something like `useCache`?

Reply:
hmmm could be, feels like cache is less descriptive of what it is, since not clear what kind of cache? what its rules? could it be out of date? when its really just a http api call that happens to also have a cache. Cache feels like it would be in support of orbitdb as well instead of just around it. But idk `api` alone is not super useful. But zooming out, if we consider that api overall a cache of orbitdb, which I guess we have, then that wording would usefully indicate what its purpose is.

Reply:
Idk, I feel like api is way to generic. Is `useCacheService` better?

Reply:
yep that works for me, lets do 'useCacheService', I think service is a good distinction